openshift/release/release-notes.md

```diff
@@ -0,0 +1,172 @@
+
+## Misc
+
+* :hammer: Update roadmap for 2021 (#1053)
+* :hammer: Add links to versioned docs for v0.13.0 (#1058)
+* :hammer: Updating to latest version of plumbing (#1064)
+* :hammer: refactor: move list of allowed ResourceTemplate types to its own package (#1068)
+* :hammer: Resolving build test issues in PRs (#1078)
+* :hammer: Migrate from test builders to structs (#1080)
+* :hammer: Bump CEL version to latest version. (#1088)
+
+The CEL interceptor is updated to the latest version of CEL, this brings in the CEL base64 decoding functions.
+These are documented here https://github.com/google/cel-go/tree/master/ext#base64decode
+**Note:** the CEL base64 decoding function decodes to a CEL byte array, which is not a string, it uses the b'hello' syntax to represent a string of bytes "hello".
+The triggers `decodeb64` CEL function is now deprecated in favour of the upstream encoders package, you can replace `body.value.decodeb64` with `string(base64.decode(body.value))`
+
+* :hammer: Bump pipeline to 0.24.1 and knative pkg to 0.22 (#1091)
+
+
+## Docs
+
+* :book: Add the new `Interceptors` Page (#1035)
+* :book: Rewrite the `ClusterInterceptors` page for clarity and flow (#1048)
+* :book: Capitalization consistency for resourcetemplates (#1050)
+* :book: Update README.md and Release Cheat Sheet (#1051)
+* :book: Update install documentation (#1054)
+* :book: Rewrite the `EventListeners` Page (#1060)
+* :book: Delete obsolete page exposing-eventlisteners.md (#1066)
+* :book: Update link to interceptors doc (#1073)
+* :book: Add link between cluster interceptor docs and interceptor docs (#1074)
+
+# Tekton Triggers v0.13.0
+
+## Features
+
+* :sparkles: Adding label selector for triggers [#970](https://github.com/tektoncd/triggers/pull/970)
+
+    Users can now configure triggers for an EventListener using labels
+
+*  :sparkles:  Added URL field to WebhookInterceptor (#1002)
+
+    Users can now configure webhook interceptors by referencing a URL
+
+*  :sparkles:  Add ClusterInterceptor CRD for registering interceptors (#960)
+
+    A new CRD called ClusterInterceptor has been added that allows for users to register new pluggable Interceptor types
+
+*  :sparkles:  Add API to configure Interceptors from a Trigger (#1001)
+    
+    Trigger spec authors can now configure interceptors using a new API that includes  a `ref` field to refer to a ClusterInterceptor, and 
+    a `params` field to add parameters to pass on to the interceptor for processing
+
+*  :sparkles:  Migrate core interceptors to use InterceptorType CRD (#976)
+
+    The four bundled interceptors (CEL, GitHub, GitLab, BitBucket) are now implemented using the new ClusterInterceptor CRD
+
+*  :sparkles:   Migrate core interceptors to new format (#1029)
+
+     Any new Triggers created using the old style syntax for core interceptors is now automatically switched to the new refs/params 
+     based syntax.
```

Comment:
@khrm How about combining all these into single sentence and linking those PR fixes

/cc @Preeticp @sounix000

Reply:
Agreed.

Reply:
@Preeticp @sounix000 could you take care of this while adding doc